cfgov/preprocessed/js/modules/util/EventObserver.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * EventObserver
+ * @class
+ *
+ * @classdesc Used for creating an object
+ *   that can be used to dispatch and listen to custom events.
+ * @returns {Object} An EventObserver instance.
+ */
+function EventObserver() {
+
+  // The events registered on this instance.
+  var _events = {};
+
+  /**
+   * Register an event listener.
+   * @param {string} evt The event name to listen for.
+   * @param {Function} callback The function called when the event has fired.
+   * @returns {Object} The instance this EventObserver instance is decorating.
+   */
+  function addEventListener( evt, callback ) {
+    if ( _events.hasOwnProperty( evt ) ) {
+      _events[evt].push( callback );
+    } else {
+      _events[evt] = [ callback ];
```

Comment:
I just think it should be consistent, and it's odd that the `[ callback ]` array that immediately follows it has spaces but this doesn't